QuickReference.ipynb

```diff
@@ -0,0 +1,402 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 0. Setup\n",
+    "Please make sure your environment is set up according to the instructions here: https://github.com/NASA-NAVO/aas_workshop_2020_winter/blob/master/00_SETUP.md\n",
+    "\n",
+    "Ensure you have the latest version of the workshop material by updating your environment:\n",
+    "TBD"
```

Comment:
maybe this could just be `git stash` followed by `git pull` followed by `git stash pop`, if the user has modified notebooks.

Reply:
I'll play with that, and set the final words on Saturday.